recurly/__init__.py

```diff
@@ -1504,6 +1504,39 @@ class CreditPayment(Resource):
         'voided_at',
     )
 
+
+class ExportDate(Resource):
+    nodename = 'export_date'
+    collection_path = 'export_dates'
+
+    attributes = ('date', 'export_files')
```

Comment:
Nit: Can you please format this the way the rest of the file formats the `attributes` tuple?
```python
attributes = (
    'date',
    'export_files'
)
```

Reply:
Sure, changed in this commit https://github.com/recurly/recurly-client-python/pull/288/commits/1d05a25c3d3fca164886e9c26bd949ce7ce5af44